src/js/views/patients/view/view_views.js

```diff
@@ -106,7 +108,15 @@ const ListView = CollectionView.extend({
   },
 });
 
+const GroupsDropList = Droplist.extend({
+  template: hbs`{{fas "filter"}} {{ name }}{{#unless name}}All Groups{{/unless}} {{far "angle-down"}}`,
```

Comment:
I don't understand the `#unless` here.  When is there not a selected?  it looks to me that `_getSelectedGroup` should always return a value, and if it doesn't it's malformed data and not "All Groups"

Reply:
yeah, looks like I don't need it? I think the last iteration of the object passed in here fixed it (maybe removing the template from the picklistOptions). Should be good to go then